testsuite/integration/manualmode/src/test/java/org/wildfly/test/manual/microprofile/health/MicroProfileHealthDefaultEmptyStartupTestBase.java

```diff
@@ -0,0 +1,294 @@
+                    connectionEstablished = true;
+                }
+
+            } while (!connectionEstablished);
+
+            boolean userChecksProcessed = false;
+
+            do {
+                try {
+                    checkGlobalOutcome(managementClient, "check-started", false, null);
+                } catch (InvalidHealthResponseException e) {
+                    // OK user checks are processed, check once more that we have an UP empty response
+                    userChecksProcessed = true;
+                    try {
+                        checkGlobalOutcome(managementClient, "check-started", true,
+                            SuccessfulStartupCheck.NAME);
+                    } catch (IOException | InvalidHealthResponseException ex) {
+                        throw new RuntimeException(e);
+                    }
+                } catch (IOException e) {
+                    throw new RuntimeException(e);
+                }
+            } while (!userChecksProcessed);
+        });
+
+        Map<String, String> map = getVMArgMap(DISABLE_DEFAULT_PROCEDURES_PROPERTY);
+        containerController.start(CONTAINER_NAME, map);
+
+        try {
+            testResultFuture.get(1, TimeUnit.MINUTES);
+        } finally {
+            deployer.undeploy(MICRO_PROFILE_HEALTH_APPLICATION_WITH_SUCCESSFUL_STARTUP_TEST_BASE);
+            containerController.stop(CONTAINER_NAME);
+        }
+
+    }
+
+    @Test
+    @InSequence(4)
+    public void testApplicationStartupWithoutDeploymentWithEmptyStartupSetToUP() throws Exception {
+
+        final CompletableFuture<Void> testResultFuture = CompletableFuture.runAsync(() -> {
+            boolean connectionEstablished = false;
+
+            do {
+                try {
+                    checkGlobalOutcome(managementClient, "check-started", true, null);
+                    connectionEstablished = true;
+                } catch (ConnectException e) {
+                    // OK, server is not started yet
+                } catch (IOException | InvalidHealthResponseException e) {
+                    throw new RuntimeException(e);
+                }
+
+            } while (!connectionEstablished);
+
+            try {
+                checkGlobalOutcome(managementClient, "check-started", true, null);
+            } catch (IOException | InvalidHealthResponseException e) {
+                throw new RuntimeException(e);
+            }
+        });
+
+        Map<String, String> map = getVMArgMap(String.format("%s %s=UP",
+            DISABLE_DEFAULT_PROCEDURES_PROPERTY, DEFAULT_STARTUP_EMPTY_RESPONSE_PROPERTY));
+        containerController.start(CONTAINER_NAME, map);
+
+        try {
+            testResultFuture.get(1, TimeUnit.MINUTES);
+        } finally {
+            containerController.stop(CONTAINER_NAME);
+        }
+    }
+
+    private Map<String, String> getVMArgMap(String vmArgs) {
+        Map<String, String> map = new HashMap<>();
+        map.put("javaVmArguments", vmArgs);
```

Comment:
@xstefank Perhaps ^^^ is the problem. My interpretation of this is it is overriding the arquillian.xml settings, i.e. https://github.com/wildfly/wildfly/pull/14400/files#diff-b9b946d9ef3b166391dbd738fb6cb326827f0af3eedef26edd7334dbb58f40ddR483

I'm not sure exactly how that breaks things on the CI server but I figure it's somehow because the -Dmaven.repo.local=${settings.repository} that ends up in ${server.jvm.args} is no longer used. Probably locally that does not matter because a local system ends up defaulting back to ~/.m2/repository, but on the CI server it's not so simple and that -Dmaven.repo.local is important.

The xerces jar is definitely present in the maven repo the CI runs are using; if not all sorts of things would break.

@luck3y If I'm right, or wrong, please chime in. :)

Reply:
You can use the surefire plugin config in the pom to pass in the standard javaVmArguments value via a system property and then append your desired test values to that.

Reply:
@bstansberry @xstefank and I were discussing this earlier, and I'd narrowed it down to the maven localrepo settings not being propagated in  when the server is started, and since its a thin server it doesn't point to the correct maven repository and fails. I think you just found the reason why these tests fail vs other similar ones pass though :) The maven repo is passed in as part of jvmArgs.

Reply:
thanks, indeed I think you're correct. This might indeed be overriding the value. I'll try tomorrow.

Reply:
adjusted, let's see if CI passes. @bstansberry I wanted to ask if this is a correct approach to pass MP Config values to the starting server. Do we have any other similar scenario?

Reply:
@xstefank CI hasn't completed but I checked a couple jobs and these tests passed on those. 👍 

Re the approach, it seems ok. There are places like https://github.com/wildfly/wildfly/blob/main/testsuite/integration/microprofile/pom.xml#L314 where the test setup is driving MP Config, but they are static, i.e. like that one a property can be set and it's fine that it's set for all tests.